docs/operator-guide/configuration.md

```diff
@@ -112,7 +112,7 @@ provides links to the relevant documentation for each.
 
 | Service |
 | ------- |
-| [cloudapi](https://github.com/joyent/sdc-cloudapi/blob/master/docs/admin.restdown#L40) |
+| [cloudapi](https://github.com/joyent/sdc-cloudapi/blob/master/docs/index.md) |
```

Comment:
Thanks for the PR!

Actually this should be 'admin.md#sapi-configuration'. Did you want to update the PR for that?